src/Microsoft.Framework.PackageManager/Restore/LockFileFormat.cs

```diff
@@ -59,6 +59,17 @@ public LockFile Read(Stream stream)
 
         public void Write(string filePath, LockFile lockFile)
         {
+            // Make sure that if the lock file exists, it is not readonly
+            if (File.Exists(filePath))
+            {
+                FileAttributes attributes = File.GetAttributes(filePath);
```

Comment:
It can fail if you don't have permissions but we should let it fail. Let the callers decide if they want to handle the exception or not. In the two cases in which we use it, we want it to fail because there is nothing we can do if we cannot remove the readonly attribute